### ISSUE TITLE
Blank page on `/status`

### ISSUE THREAD

**Issue by Starfight** [YunoHost-Apps/uptime-kuma_ynh#13]
### Describe the bug

*All page like dashboard and settings work well, but when I click on `/status`, it open a new blank page. In developer console, I see some API request error on 404. But if I change on `/status/default` or ` /manage-status-page`, these 404 errors disappear but page stay empty.*

### Context

- Hardware: *VPS bought online*
- YunoHost version: *4.3.6.3*
- I have access to my server: *Through SSH*
- Are you in a special context or did you perform some particular tweaking on your YunoHost instance?: *no*
- Using, or trying to install package version/branch: *1.13.1*

### Steps to reproduce

   1. *Go to uptime-kuma host, add a new probe*
   2. *Click on Status*
   4. *See error*

### Expected behavior

*I expect to see status page to manage a new one.*

### Logs

*No errors in logs.*

**Comment by ketsapiwiq** [comment]
Hi!
I was wondering, did you by any chance try to move the app's domain or URL?
Or were you maybe already using the domain of uptime-kuma before installing it?

**Comment by Starfight** [comment]
@ketsapiwiq As expected by the app, I create a custom subdomain for uptime-kuma before installation.

**Comment by ketsapiwiq** [comment]
Can you post an extract of the web console (`Ctrl+Shift+I` on Chrome or Firefox) please?

**Comment by Starfight** [comment]
```
XHR GET https://mydomain.fr/api/status-page/incident [[HTTP/2 404 Not Found 45ms]]()
XHR GET https://mydomain.fr/api/status-page/config [[HTTP/2 404 Not Found 44ms]]()
XHR GET https://mydomain.fr/api/status-page/monitor-list [[HTTP/2 404 Not Found 45ms]]()
XHR GET https://mydomain.fr/api/status-page/heartbeat [[HTTP/2 404 Not Found 45ms]]()
```
Then 4 `Uncaught (in promise) Error: Request failed with status code 404`

**Comment by ketsapiwiq** [comment]
Thanks a lot.
Sorry I've not had much time to check this.
Apparently the update to v1.13.2 fixes this, we'll merge it when the CI tests it.

**Comment by ketsapiwiq** [comment]
It got merged, I hope it fixed your problem!

Strangely it didn't fix mine (I had the same problem) with 1.13.2 and got lucky with 1.14.1, but the it was the `/admin` page that got blank, so I'll investigate a bit more just in case it's because of 1.14.1.

**Comment by Starfight** [comment]
Perfect, it was not solved with v1.13.2 but with 1.14.1 it works well. Thank you.
